### PR TITLE
Fixes #6

### LINKED ISSUE #6: Refresh landing page
Here's my first take at a rethink of the landing page content after our meeting:

redesign

Focus less on the text (we can easily change that). I'd like to figure out if folks feel a layout like this or one like it would be more useful than what we have now.

We could easily expand the later sections to include more features/breakdowns. I kind of love how clean the http://babeljs.io/ site is and tried to draw some inspiration from it.

Shared the Sketch file for my silly attempt with @gauntface

### PR DESCRIPTION
**Pull request by gauntface**
Adds comments from #6

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
-# Installation
+# Getting Started
```
Comment (jpmedley) [inline src/index.md]: The navigation bar says "Get Started". Can you make this consistent?

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/index.md (hunk 1 above)
Comment (gauntface) [reply src/index.md]: Do you want getting started or get started?

Comment (jpmedley) [reply src/index.md]: Oh, sorry. 'Get Started'. I had this in another PR https://github.com/GoogleChrome/workbox-microsite/pull/15

**Review by jpmedley — APPROVED** [review]

**Comment by addyosmani** [comment]
lgtm

**Comment by jeffposnick** [comment]
👍

**Pull request merged**